devices/motorola-addison/firmware/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, runCommandNoCC, fetchFromGitHub, fetchurl }:
+
+let
+  # FIXME: somehow get the user to provide the contents of the "modem" partition
+  #     sudo blkid | grep -i '"modem"'
+  #     /dev/mmcblk0p19: UUID="57f8f4bc-abf4-655f-bf67-946fc0f9f25b" BLOCK_SIZE="4096" TYPE="ext4" PARTLABEL="modem" PARTUUID="e9018b02-0068-cb79-4fb4-b0b9027c4c80"
+
+  # Contents of the "modem" partition.
+  modem = ./modem;
```

Comment:
**This** is where I need to figure out how to let the user provide the content of the `modem` partition.